dev_notes/Penetration Test Assesment.md

```diff
@@ -0,0 +1,90 @@
+## Risk Identification
+1. Identify assets (e.g. web application)
+    1. Web application
+    2. Api
+    3. Docker Hub
+    4. Github repository 
+    5. Digital Ocean
+    6. Grafana / Prometheus / Kibana
+2. Identify threat sources 
+    1. SQL injection 
+    2. Cross-Site Scripting (XSS) attacks
+    3. Denial of Service (DoS) attacks
+    4. Password leakage
+    5. Exposure of hard-coded passwords in public repositories (e.g., GitHub) 
+    6. Unused open ports 
+    7. Administrator’s passwords and their strength on all relevant platforms
+    8. Vulnerabilities in frameworks or docker images used
```

Comment:
maybe change the order of this from: "docker images used" to: "used docker images"